src/main/clojure/jarl/parser.clj

```diff
@@ -100,65 +81,65 @@
     (assoc state :result-set (conj result-set value))))
 
 (defn make-ArrayAppendStmt [stmt-info]
-  (log-debug "making ArrayAppendStmt stmt")
+  (log/debugf "making ArrayAppendStmt stmt")
```

Comment:
Is there any point in using the `f` version of the logger functions if no args are provided?

Reply:
No. I was just a bit over-zealous with the find/replace 😄 .